lisa/node.py

```diff
@@ -113,6 +113,7 @@ def __init__(
         self.capture_azure_information: bool = False
         self.capture_kernel_config: bool = False
         self.has_checked_bash_prompt: bool = False
+        self.capture_vm_information: bool = True
```

Comment:
It doesn't need to save on node level. Keep it in platform runbook should be enough.